.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,46 @@
+name: Docker Build
+
+on:
+  push:
+    branches: [ "master" ]
```

Comment:
Runs only when we merge to master. But I already did a test in pull requests and works fine, see [successful run here](https://github.com/eth-act/zkevm-benchmark-workload/actions/runs/16008858949/job/45161761467?pr=99).